### PR TITLE
Content create artifacts performence- test files

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Test files for - https://github.com/demisto/demisto-sdk/pull/625

### CODE REVIEW

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample01/Classifiers/classifier-sample_new_CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (yaakovi) [inline demisto_sdk/tests/test_files/content_slim/Packs/Sample01/Classifiers/classifier-sample_new_CHANGELOG.md:1]: can be removed

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Layouts/layout-sample2_packs.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Layouts/layout-sample2_packs.json:1]: `layout` is only for 5.5 and below.
should be `layoutscontainer`

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Layouts/layout-sample2_packs_new.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Layouts/layout-sample2_packs_new.json:1]: you can remove this file as `layout` file is not supported for both `packs` and `new`

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample02/IndicatorFields/incidentfield-sample2.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/tests/test_files/content_slim/Packs/Sample02/IndicatorFields/incidentfield-sample2.json:1]: any reason this is a full file?

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Classifiers/classifier-sample2_new_README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Classifiers/classifier-sample2_new_README.md:1]: was that supposed to be a README or a JSON file?

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Classifiers/classifier-sample2_packs_new.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Classifiers/classifier-sample2_packs_new.json:1]: you can remove this file, classifier file will not go in both `packs` and `new`

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample01/Layouts/layout-sample_packs_new.json [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/tests/test_files/content_slim/Packs/Sample01/Layouts/layout-sample_packs_new.json:1]: same

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample01/Layouts/layout-sample_packs.json [hunk 8]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/tests/test_files/content_slim/Packs/Sample01/Layouts/layout-sample_packs.json:1]: same

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample02/Layouts/layout-sample2_packs_new.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample02/Layouts/layout-sample2_packs_new.json:1]: same

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample02/Layouts/layout-sample2_packs.json [hunk 10]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample02/Layouts/layout-sample2_packs.json:1]: same

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample01/Layouts/layout-sample_packs_new.json [hunk 11]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample01/Layouts/layout-sample_packs_new.json:1]: same

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample01/Layouts/layout-sample_packs.json [hunk 12]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample01/Layouts/layout-sample_packs.json:1]: same

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_new/layout-sample2_packs_new.json [hunk 13]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_new/layout-sample2_packs_new.json:1]: same

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_new/layout-sample_packs_new.json [hunk 14]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yaakovi) [inline demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_new/layout-sample_packs_new.json:1]: same

**Comment by GalRabin** [comment]
Regarding CHANGELOG and readme, I need them in order to validate that I don't collect them.

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample02/IndicatorFields/incidentfield-sample2.json (hunk 4 above)
Comment (GalRabin) [reply demisto_sdk/tests/test_files/content_slim/Packs/Sample02/IndicatorFields/incidentfield-sample2.json:1]: Yes, Findtype knows how to detect it only by the file content, due to the fact that it starts with the incident field.

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Layouts/layout-sample2_packs.json (hunk 2 above)
Comment (GalRabin) [reply demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Layouts/layout-sample2_packs.json]: Changed

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Classifiers/classifier-sample2_packs_new.json (hunk 6 above)
Comment (GalRabin) [reply demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Classifiers/classifier-sample2_packs_new.json]: Removed

File: demisto_sdk/tests/test_files/content_slim/Packs/Sample01/Layouts/layout-sample_packs_new.json (hunk 7 above)
Comment (GalRabin) [reply demisto_sdk/tests/test_files/content_slim/Packs/Sample01/Layouts/layout-sample_packs_new.json]: Changed

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample02/Layouts/layout-sample2_packs_new.json (hunk 9 above)
Comment (GalRabin) [reply demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample02/Layouts/layout-sample2_packs_new.json]: Changed

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample01/Layouts/layout-sample_packs_new.json (hunk 11 above)
Comment (GalRabin) [reply demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample01/Layouts/layout-sample_packs_new.json]: Changed

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample01/Layouts/layout-sample_packs.json (hunk 12 above)
Comment (GalRabin) [reply demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample01/Layouts/layout-sample_packs.json]: Changed

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample02/Layouts/layout-sample2_packs.json (hunk 10 above)
Comment (GalRabin) [reply demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_packs/Sample02/Layouts/layout-sample2_packs.json]: Changed

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_new/layout-sample2_packs_new.json (hunk 13 above)
Comment (GalRabin) [reply demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_new/layout-sample2_packs_new.json]: Changed

File: demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_new/layout-sample_packs_new.json (hunk 14 above)
Comment (GalRabin) [reply demisto_sdk/commands/create_artifacts/tests/content_artifacts_creator_test/test_create_content_artifacts/content_private_expected_artifact/content_new/layout-sample_packs_new.json]: Changed

**Comment by GalRabin** [comment]
I modified as required and check that tests still passing, So we will not need new PR

**Pull request merged**